net/node/localnode.go

```diff
@@ -63,7 +63,7 @@ func (localNode *LocalNode) MarshalJSON() ([]byte, error) {
 	}
 
 	out["height"] = localNode.GetHeight()
-	out["uptime"] = time.Since(localNode.startTime).Truncate(time.Second).String()
+	out["uptime"] = time.Since(localNode.startTime).Truncate(time.Second).Seconds()
```

Comment:
Any incompatible from client or SDK side?

Reply:
Nope, no one is using it atm